consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/network/IbftNetworkPeers.java

```diff
@@ -57,6 +54,13 @@ public void multicastToValidators(final MessageData message) {
     sendMessageToSpecificAddresses(validators, message);
   }
 
+  public void multicastToValidatorsExcept(
+      final MessageData message, final Collection<Address> exceptAddresses) {
+    final Collection<Address> validators = validatorProvider.getValidators();
+    validators.removeIf(exceptAddresses::contains);
```

Comment:
Depending on the rest of the context it may be even better to have `sendMessageToSpecificAddresses` accept a `Stream<Address>` instead of a `Collection<Address>` and then you can just use `validatorProvider.getValidators().stream().filter(exceptAddresses::contains)` and avoid any need to copy and modify.

Reply:
fixed